insights/parsers/tests/test_du_usage.py

```diff
@@ -0,0 +1,15 @@
+from insights.parsers.du_usage import DiskUsage
+from insights.tests import context_wrap
+
+disk_content = """
+COMMAND> du -sh /var/lib/mongodb
+
+22G     /var/lib/mongodb
+""".strip()
+
+
+def test_du_disk_usage():
+    context = context_wrap(disk_content)
+    dbdisk_info = DiskUsage(context)
+
+    assert dbdisk_info.get_size('/var/lib/mongodb') == "22G"
```

Comment:
You need a lot more tests:
* Tests of input of multiple directories.
* Tests of handling paths with spaces.
* Tests of handling other invalid inputs.
* Tests of what should be returned when retrieving directories that weren't read.
* Tests of other access methods - access via `data` attribute and via the LegacyItemAccess methods.